scene/gui/color_picker.cpp

```diff
@@ -153,8 +159,40 @@ void ColorPicker::_notification(int p_what) {
 			}
 		} break;
 
+		case NOTIFICATION_FOCUS_ENTER:
+		case NOTIFICATION_FOCUS_EXIT: {
+			cursor_editing = false;
+		} break;
+
 		case NOTIFICATION_INTERNAL_PROCESS: {
 			if (!is_picking_color) {
+				Input *input = Input::get_singleton();
+				bool is_picker_focused = uv_edit->has_focus() || wheel_uv->has_focus() || wheel->has_focus();
+
+				if (input->is_action_just_released("ui_left") ||
+						input->is_action_just_released("ui_right") ||
+						input->is_action_just_released("ui_up") ||
+						input->is_action_just_released("ui_down")) {
+					gamepad_event_delay_ms = DEFAULT_GAMEPAD_EVENT_DELAY_MS;
+					echo_multiplier = 1;
+					accept_event();
+					set_process_internal(false);
+					return;
+						}
+
+				gamepad_event_delay_ms -= get_process_delta_time();
+				if (gamepad_event_delay_ms <= 0) {
+					gamepad_event_delay_ms = GAMEPAD_EVENT_REPEAT_RATE_MS + gamepad_event_delay_ms;
+					// Treat any input from joypad axis as -1, 0, or 1, as the value is added to Vector2i and would be lost
```

Comment:
```suggestion
					// Treat any input from joypad axis as -1, 0, or 1, as the value is added to Vector2i and would be lost.
```

Reply:
Thanks for all your suggestions! I updated this right away and I see those changes [here](https://github.com/godotengine/godot/compare/c90efdb83d9a321aba97271dc0893ff0005d08e8..80de4db26529d5f8712bd268f2f15c6292e2d80d), but it looks like GitHub didn't update this everywhere. It should show all your suggestions as outdated but it doesn't, I don't know why.

Reply:
Weird, might be because of your style fixes changing code indentation

Reply:
Ok, everything looks fine now. I realized that others were marked as outdated after few minutes and this one was my ommission, because I have the same comment in two places and I missed one. It's fixed now.